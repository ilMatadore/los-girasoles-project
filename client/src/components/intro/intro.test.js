import React from 'react';
import { shallow } from 'enzyme';
import Intro from './intro.component';

it('Expect to render Intro component', () => {
    expect(shallow(<Intro />)).toMatchSnapshot();
})