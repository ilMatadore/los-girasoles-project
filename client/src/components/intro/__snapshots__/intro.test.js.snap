// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect to render Intro component 1`] = `
<div
  className="intro-container"
>
  <p
    className="intro-text"
    data-aos="zoom-out"
  >
    Productos Org√°nicos de Nuestra Tierra a tu Hogar
  </p>
  <a
    className="intro-button"
    href="#canastas"
  >
    HACE TU PEDIDO
  </a>
</div>
`;
